mne/tests/test_coreg.py

```diff
@@ -279,3 +279,32 @@ def test_get_mni_fiducials():
     fids_est = np.array([f['r'] for f in fids_est])
     dists = np.linalg.norm(fids - fids_est, axis=-1) * 1000.  # -> mm
     assert (dists < 8).all(), dists
+
+
+@testing.requires_testing_data
+def test_coregistration():
+    """Test automated coregistration."""
+    fname_raw = op.join(op.dirname(__file__), '..', 'io',
+                        'tests', 'data', 'test_raw.fif')
+    subject = 'sample'
+    subjects_dir = os.path.join(data_path, 'subjects')
+    info = read_info(fname_raw)
+    coreg = Coregistration(info, subject=subject, subjects_dir=subjects_dir)
+    assert np.allclose(coreg._last_parameters, coreg._parameters)
+    default_params = list(coreg._default_parameters)
+    coreg.set_rotation(default_params[:3])
+    coreg.set_translation(default_params[3:6])
+    coreg.set_scale(default_params[6:9])
+    coreg.set_grow_hair(0.)
+    coreg.fit_fiducials()
+    assert not np.allclose(coreg._parameters, default_params)
+    assert coreg._extra_points_filter is None
+    coreg.omit_hsp_points(distance=-1)
+    coreg.omit_hsp_points(distance=5. / 1000)
+    assert coreg._extra_points_filter is not None
+    coreg.fit_icp()
+    assert isinstance(coreg.trans, Transform)
+    errs_icp = coreg.compute_dig_head_distances()
+    assert np.median(errs_icp * 1000) < 4
+    coreg.reset()
+    assert np.allclose(coreg._parameters, default_params)
```

Comment:
It would be good to load the actual `-trans.fif` and assert that the translation and rotation (using `angle_between_quats`) is sufficiently small

Reply:
so my idea is:

```
assert_allclose(_angle_between_quats(rot_to_quat(coreg._rotation), X)
```

where `X` is the quat obtained from the rot and trans matrices in the `-trans.fif` file but I don't know how to extract those.

Reply:
After an offline discussion with @agramfort, I tried b6abb03 to test the angle between trans.